erpnext/public/js/controllers/accounts.js

```diff
@@ -146,18 +146,21 @@ cur_frm.cscript.account_head = function(doc, cdt, cdn) {
 	if(!d.charge_type && d.account_head){
 		frappe.msgprint(__("Please select Charge Type first"));
 		frappe.model.set_value(cdt, cdn, "account_head", "");
-	} else if(d.account_head && d.charge_type!=="Actual") {
+	} else if(d.account_head) {
 		frappe.call({
 			type:"GET",
 			method: "erpnext.controllers.accounts_controller.get_tax_rate",
 			args: {"account_head":d.account_head},
 			callback: function(r) {
-				frappe.model.set_value(cdt, cdn, "rate", r.message.tax_rate || 0);
-				frappe.model.set_value(cdt, cdn, "description", r.message.account_name);
+				if(d.charge_type!=="Actual"){
+					frappe.model.set_value(cdt, cdn, "rate", r.message.tax_rate || 0);
+					frappe.model.set_value(cdt, cdn, "description", r.message.account_name);
+				}
+				else if(d.charge_type == 'Actual'){
+					frappe.model.set_value(cdt, cdn, "description", r.message.account_name);
+				}
```

Comment:
code can be simplified by
```suggestion
				if (d.charge_type !== "Actual") {
					frappe.model.set_value(cdt, cdn, "rate", cint(r.message.tax_rate));
				}
				frappe.model.set_value(cdt, cdn, "description", r.message.account_name);
```
|

Also, please make sure if we are not introducing [this bug again](https://github.com/frappe/erpnext/pull/13843)